fudge-ratio/src/main/webapp/style.css

```diff
@@ -1 +1,14 @@
-/* This is a css page for the index.html */
\ No newline at end of file
+/* This is a css page for the index.html */
+
+.container {
+  background: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
```

Comment:
also, thanks for using vh, very interesting

Reply:
No problem!